recipes/libavrocpp/1.10.1/conanfile.py

```diff
@@ -0,0 +1,100 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+
+
+class LibavrocppConan(ConanFile):
+    name = "libavrocpp"
+    version = "1.10.1"
+    license = "Apache License 2.0"
```

Comment:
We only used the SPDX short identifiers

```suggestion
    license = "Apache-2.0"
```